src/app/courses/courses.component.html

```diff
@@ -41,13 +41,13 @@ <h3 class="header">{{element.courseTitle}}</h3>
         </button>
         <mat-cell>
           <mat-menu #resourceMenu="matMenu">
-            <a mat-menu-item routerLink="add">
+            <a *ngIf="element.resign" mat-menu-item routerLink="add">
               <mat-icon>lock_open</mat-icon>
               <span i18n>Open</span>
             </a>
-            <a mat-menu-item routerLink="add">
-              <mat-icon>clear</mat-icon>
-              <span i18n>Resign</span>
+            <a mat-menu-item (click)="toggleResign(element,element._id)">
```

Comment:
**element** contains _id,_rev and other fields . So , do we need two parameters here ?